src/main/java/yilia/exception/YiliaException.java

```diff
@@ -0,0 +1,16 @@
+package yilia.exception;
```

Comment:
you have many exception classes, maybe you can consider letting them inherit from one exception parent class.